packages/frontend/model/clean.ts

```diff
@@ -3,10 +3,16 @@ import createDOMPurify from 'dompurify';
 import { JSDOM } from 'jsdom';
 import { minify } from 'html-minifier';
 
+// We don't represent lists in incopy, so things will just come across with bullet characters.
+// These may also be used for emphasis, so bullet characters don't mean list.
+const bigBullets = (s: string) =>
+    s.replace(/•/g, '<span class="bullet">&bull;</span>');
+
```

Comment:
Do you fancy adding a test for this -- or just moving the one from `extract-capi.test.ts`?